scss/skin/_default.scss

```diff
@@ -47,12 +47,14 @@
 
   // "full window" style, if fullscreen is not supported natively
   &.oo-fullscreen {
-    position:fixed;
-    overflow: hidden;
-    top:0;
-    left:0;
-    width:100%;
-    height:100%;
+    width: 100%!important;
+    height: 100%!important;
+    top: 0px;
+    bottom: 0px;
```

Comment:
Should bottom be 100%? Having both top and bottom at 0 would create a flat line if height wasn't 100%/

Reply:
we refactored css